docs/source/model_design/risk_factors.rst

```diff
@@ -134,8 +134,131 @@ Definitions
+[GBD-2017-Risk-Appendix]_
+
+Recall from the `causality section <What is a causal relationship?_>`_ that
+counterfactual analysis is used to describe the causal relationship between a
+risk factor and an outcome. **The TMRED is a particular choice of counterfactual
+exposure distribution** used for the causal attribution of disease burden to a
+given risk factor in a population (see `Population Attributable Fraction
+(PAF)`_). Other choices of counterfactual include the *plausible* minimum risk,
+*feasible* minimum risk, and *cost-effective* minimum risk, each of which can
+obviously depend on specific attributes of the population under consideration.
+On the other hand, Murray et al. state
+[Comparative-quantification-health-risks]_:
+
+  Biological principles as well as considerations of equity would necessitate
+  that, **although the exposure distribution for theoretical minimum risk may
+  depend on age and sex, it should in general be independent of geographical
+  region or population.**
+
+However, they go on to add:
+
+  Exceptions to this are however unavoidable. An example would be the case of
+  alcohol consumption, which in limited quantities and certain patterns, has
+  beneficial effects on cardiovascular mortality, but is always harmful for
+  other diseases such as cancers and accidents. In this case, the composition of
+  the causes of death as well as drinking patterns in a region would determine
+  the theoretical minimum distribution.
+
+.. todo::
+
+  Is there an updated/better example of TMRED depending on population, since the
+  latest research says that that there is no safe amount of alcohol?
+
+The smoking example `above <Theoretical Minimum Risk Exposure Level/Distribution
+(TMREL/D)_>`_ illustrates two features of the TMREL that are typical of many
+risk factors:
+
+1. We imagine that everyone in the population has the same TMREL
+2. The TMREL is *zero* or *no exposure*
+
+However, neither of these conditions is necessary. In some cases, particularly
+for continuous risk exposure variables, the TMREL may be a nonzero exposure
+level. Moreover, there may be multiple TMRELs experienced by different members
+of the population. For example, in GBD 2017 [GBD-2017-Risk-Appendix]_:
+
+1.  The TMREL for radon exposure is taken to be 10 `Bq
+    <https://en.wikipedia.org/wiki/Becquerel>`_/m\ :superscript:`3`, which is
+    equivalent to the average outdoor concentration of radon [ICRP]_.
+2.  The :ref:`Low Birth Weight and Short Gestation <2017_risk_lbwsg>` risk
+    factor has multiple TMREL categories since healthy babies have many
+    different birth weights and gestational ages.
+
+These examples illustrate some complexities in defining the TMREL and TMRED for
+a given risk factor. For continuous risk exposure variables --- such as radon
+exposure, or hemoglobin concentration, or systolic blood pressure --- it may be
+impossible to define a single TMREL for the population, as we expect different
```

Comment:
Is there anything in the literature about distributions of TMREDs (or TMRELs for an individual?)